schematic/schemas/data_model_jsonld.py

```diff
@@ -0,0 +1,284 @@
+                                            if (isinstance(template[rel_vals['jsonld_key']], list) and
+                                                node_2_id not in template[rel_vals['jsonld_key']]):
+                                                template[rel_vals['jsonld_key']].append(node_2_id)
+                                            else:
+                                                template[rel_vals['jsonld_key']] == node_2
+            # Fill in node value information
+            else:               
+                node_label = rel_vals['node_label']
+                
+                # Get recorded info for current node, and the attribute type
+                node_info = nx.get_node_attributes(self.graph, node_label)[node]
+                
+                # Add this information to the template
+                template[rel_vals['jsonld_key']] =  node_info
+        
+        # Clean up template
+        template = self.clean_template(template=template,
+                                       data_model_relationships=data_model_relationships,
+                                       )
+        # Reorder lists based on weights:
+        template = self.reorder_template_entries(template=template,)
+
+        # Add contexts to certain values
+        template = self.add_contexts_to_entries(template=template,)
+
+        return template
+
+    def add_contexts_to_entries(self, template:dict) -> dict:
+        """
+        Args:
+            template, dict: JSONLD template that has been filled up to the current node, with information
+        Returns:
+            template, dict: JSONLD template where contexts have been added back to certain values.
+        Note: This will likely need to be modified when Contexts are truly added to the model
+        """
+        for jsonld_key, entry in template.items():
+            try:
+                # Retrieve the relationships key using the jsonld_key
+                key= [k for k, v in self.rel_dict.items() if jsonld_key == v['jsonld_key']][0]
+            except:
+                continue
+            # If the current relationship can be defined with a 'node_attr_dict'
+            if 'node_attr_dict' in self.rel_dict[key].keys():
+                try:
+                    # if possible pull standard function to get node information
+                    rel_func = self.rel_dict[key]['node_attr_dict']['standard']
+                except:
+                    # if not pull default function to get node information
+                    rel_func = self.rel_dict[key]['node_attr_dict']['default']
+
+                # Add appropritae contexts that have been removed in previous steps (for JSONLD) or did not exist to begin with (csv)
+                if key == 'id' and rel_func == get_label_from_display_name:
+                    template[jsonld_key] = 'bts:' + template[jsonld_key]
+                elif key == 'required' and rel_func == convert_bool_to_str:
```

Comment:
If the template value is left, then SMS would not have been stripped. Need to add a check to see if `sms` is already part of the str before trying to add it back, same for above with `bts` (in this case its an extra safety)